covidWatch.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
```

Comment:
@Oyeteju can you rename this file to `news.html`, just to make it more descriptive.